Persistence/RegistryBrowserStartPage.c

```diff
@@ -0,0 +1,36 @@
+#include <stdio.h>
+#include <windows.h>
+
```

Comment:
Do you want to add more author information here, such as a social media handle or email? Completely up to you, I realize the blog page accomplishes that to a certain degree :)

Reply:
Sure! I've went ahead and updated it with my Github username! Thanks!